pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/AdminResource.java

```diff
@@ -245,6 +245,19 @@ protected Policies getNamespacePolicies(String property, String cluster, String
         }
     }
 
+    protected LocalPolicies getNamespaceLocalPolicies(String property, String cluster, String namespace) {
```

Comment:
Though BundleData is different. It is not configuration. It is written back, by the brokers themselves, into the local ZK because we want each cluster to be able to split the bundles independently.

For rate limiting, we are talking about proper configuration.